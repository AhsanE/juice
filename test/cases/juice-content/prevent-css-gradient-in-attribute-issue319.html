<html>
<head>
    <style>
        .bg-gradient { 
            background-image: linear-gradient(to right, #fdc830, #f37335); 
        }
    </style>
</head>
<body>
    <table class="bg-gradient">
        <tr>
            <th class="bg-gradient">Lorem</th>
        </tr>
        <tr>
            <td class="bg-gradient">Ipsum</td>
        </tr>
    </table>
</body>
</html>