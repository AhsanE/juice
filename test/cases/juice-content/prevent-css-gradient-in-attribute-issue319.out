<html>
<head>
    
</head>
<body>
    <table class="bg-gradient" style="background-image: linear-gradient(to right, #fdc830, #f37335);">
        <tr>
            <th class="bg-gradient" style="background-image: linear-gradient(to right, #fdc830, #f37335);">Lorem</th>
        </tr>
        <tr>
            <td class="bg-gradient" style="background-image: linear-gradient(to right, #fdc830, #f37335);">Ipsum</td>
        </tr>
    </table>
</body>
</html>